CONTRIBUTING.md

```diff
@@ -0,0 +1,32 @@
+# Contributing to Nit
+
+Thank you for your interest in contributing to Nit!
+We accept all forms of contribution, ranging from small example programs to improvements and additions to the libraries and posting of issues.
+
+
+## Contribution process
+
+All the contributions to the Nit language, its tools and libraries, are done through Github Pull-Requests.
+
+All the information on our contribution policy is available on our [website](http://nitlanguage.org/internal/).
+We encourage you to take a look at the sections available here for a walthrough of what is required to submit a new PR.
+
+### I'm a beginner and I'd like to program in Nit, where do I start ?
+
+If you are looking for inspiration to begin programming in Nit, we encourage you to take a look at our issues. Some of them are marked with the `newcomer` tag.
+These issues are aimed at beginners with the language, and are a way to encourage people with no experience with Nit to engage in easy, fun tasks.
```

Comment:
Already stated by the “Get Involved” page. If the wiki not clear enough, you should update it instead of paraphrasing it IMHO.